lalrpop/src/lr1/build/mod.rs

```diff
@@ -203,7 +209,8 @@ impl<'grammar, L: LookaheadBuild> LR<'grammar, L> {
             // The `nt` will be X and the `remainder` will be `z...`.
             let (nt, remainder) = match shift_symbol {
                 None => continue, // requires a reduce
-                Some((Symbol::Terminal(_), _)) => continue, // requires a shift
+                Some((Symbol::Terminal(_), _)) |
+                Some((Symbol::Error, _)) => continue, // requires a shift
```

Comment:
Nit: can you put `=> { \n continue; \n } \n` instead?

Otherwise, I find it hard to visually "see" the groups.